lib/resources/service.rb

```diff
@@ -587,6 +594,7 @@ def info(service_name)
         installed: true,
         running: service_running?(service),
         enabled: service_enabled?(service),
+        startmode: !service['WMI']['StartMode'].nil? ? service['WMI']['StartMode'] : nil,
```

Comment:
Seems like we don't need to check for nil - if it is nil, we assign nil, otherwise we assign whatever non-nil value we get.  It's just a cosmetic change though - i don't think this should hold up merging this PR.